torchvision/models/feature_extraction.py

```diff
@@ -0,0 +1,507 @@
+          sub-module, turn it into a "leaf module" by passing a list of
+          `leaf_modules` as one of the `tracer_kwargs` (see example below). It
+          will not be traced through, but rather, the resulting graph will
+          hold a reference to that module's forward method.
+        - Likewise, you may turn functions into leaf functions by passing a
+          list of `autowrap_functions` as one of the `tracer_kwargs` (see
+          example below).
+        - Some inbuilt Python functions can be problematic. For instance,
+          `int` will raise an error during tracing. You may wrap them in your
+          own function and then pass that in `autowrap_functions` as one of
+          the `tracer_kwargs`.
+        - Use `torch.matmul(tensor_a, tensor_b)` instead of `tensor_a @
+          tensor_b`.
+
+    For further information on FX see the
+    `torch.fx documentation <https://pytorch.org/docs/stable/fx.html>`_.
+
+    Args:
+        model (nn.Module): model on which we will extract the features
+        return_nodes (list or dict, optional): either a `List` or a `Dict`
+            containing the names (or partial names - see note above)
+            of the nodes for which the activations will be returned. If it is
+            a `Dict`, the keys are the node names, and the values
+            are the user-specified keys for the graph module's returned
+            dictionary. If it is a `List`, it is treated as a `Dict` mapping
+            node specification strings directly to output names. In the case
+            that `train_return_nodes` and `eval_return_nodes` are specified,
+            this should not be specified.
+        train_return_nodes (list or dict, optional): similar to
+            `return_nodes`. This can be used if the return nodes
+            for train mode are different than those from eval mode.
+            If this is specified, `eval_return_nodes` must also be specified,
+            and `return_nodes` should not be specified.
+        eval_return_nodes (list or dict, optional): similar to
+            `return_nodes`. This can be used if the return nodes
+            for train mode are different than those from eval mode.
+            If this is specified, `train_return_nodes` must also be specified,
+            and `return_nodes` should not be specified.
+        tracer_kwargs (dict): a dictionary of keywork arguments for
+            `NodePathTracer` (which passes them onto it's parent class
+            `torch.fx.Tracer`).
+        suppress_diff_warning (bool): whether to suppress a warning when there
+            are discrepancies between the train and eval version of the graph.
+            Defaults to False.
+
+    Examples::
+
+        >>> # Feature extraction with resnet
+        >>> model = torchvision.models.resnet18()
+        >>> # extract layer1 and layer3, giving as names `feat1` and feat2`
+        >>> model = torchvision.models._utils.build_feature_extractor(
```

Comment:
```suggestion
        >>> model = torchvision.models.feature_extraction.build_feature_extractor(
```

Also If we are to make `feature_extraction` public as a stable feature.
It would need an import in the `__init__` file.
Of course the import would then change to
```suggestion
        >>> model = torchvision.models.build_feature_extractor(
```

Reply:
@oke-aditya thanks! Sorted and will push soon